crmsh/bootstrap.py

```diff
@@ -743,36 +743,16 @@ def mkdirs_owned(dirs, mode=0o777, uid=-1, gid=-1):
 
 def init_ssh():
     """
-    Configure passwordless SSH.
+    Configure passwordless SSH with crmsh specific key.
     """
     start_service("sshd.service")
-    invoke("mkdir -m 700 -p /root/.ssh")
-    if os.path.exists("/root/.ssh/id_rsa"):
-        if _context.yes_to_all and _context.no_overwrite_sshkey or \
-                not confirm("/root/.ssh/id_rsa already exists - overwrite?"):
+    if os.path.exists(SSH_KEY_CRMSH):
+        if not confirm("{} already exists - overwrite?".format(SSH_KEY_CRMSH)):
             return
-        rmfile("/root/.ssh/id_rsa")
-    status("Generating SSH key")
-    invoke("ssh-keygen -q -f /root/.ssh/id_rsa -C 'Cluster Internal' -N ''")
-    append("/root/.ssh/id_rsa.pub", "/root/.ssh/authorized_keys")
-
-
-def init_ssh_remote():
-    """
-    Called by ha-cluster-join
-    """
-    authorized_keys_file = "/root/.ssh/authorized_keys"
-    if not os.path.exists(authorized_keys_file):
-        open(authorized_keys_file, 'w').close()
-    authkeys = open(authorized_keys_file, "r+")
-    authkeys_data = authkeys.read()
-    for key in ("id_rsa", "id_dsa", "id_ecdsa", "id_ed25519"):
-        fn = os.path.join("/root/.ssh", key)
-        if not os.path.exists(fn):
-            continue
-        keydata = open(fn + ".pub").read()
-        if keydata not in authkeys_data:
-            append(fn + ".pub", authorized_keys_file)
+        rmfile(SSH_KEY_CRMSH)
+        invoke("sed -i '/crmsh-generated/d' /root/.ssh/authorized_keys")
```

Comment:
Add this `crmsh-generated` to a constant. I have seen it in many places. It will be better if we decide to change the value for any reason.

Reply:
changed as `constants.SSH_KEY_CRMSH_TAG`